.gitlab-ci.yml

```diff
@@ -35,7 +35,7 @@ build-test:
     - mozmeao
     - aws
   script:
-    - make clean build-ci
+    - bin/build-release-image.sh
```

Comment:
I can't find this file in the repo. Maybe you forgot to `git add`?

Reply:
That is so strange. I remember creating it, but it's just gone now. I can't find a trace of it. Good catch.

Reply:
Okay. I've recreated it.